docs/how-to-run.adoc

```diff
@@ -53,6 +53,13 @@ This script requires Omicron be uninstalled, e.g., with `pfexec
 that is not the case. The script will then remove the file-based vdevs and the
 VNICs created by `create_virtual_hardware.sh`.
 
+=== Make me a certificate!
```

Comment:
Is there an analogous change for `how-to-run-simulated`?  In general, the test suite matches what `how-to-run-simulated` does so if you've updated that then things should work.

Reply:
There is not, because the implementation within Nexus checks for the existence of the certificate files before deciding whether or not to launch the HTTPS server.

For the simulated server, they won't exist, so only the HTTP server will be launched.

I definitely think this would be required before moving to "HTTPS only", but considered this PR an intermediate step.

Reply:
Mentioned in the other comment, but that doesn't seem like the right behavior.  It seems like in production, if they don't exist, that should be a fatal error.  And in development, if they do exist, we should start the HTTP server.  If this were part of configuration, then whoever's running Nexus can express their intent.  (I'm not saying we have to do all that in this PR, but if we decide that _is_ what we want, then I think this behavior is not really an intermediate step.)